### PR TITLE
adding Bag model, random Bag Generator and Payment Generator

### PR DESCRIPTION
**Pull request by Lukasz-Kawa**
I have created Bag model with Bag Random Generator and Payment generator based on the app.
Please review and check if i have not missed anything

### CODE REVIEW

**Comment by Lukasz-Kawa** [comment]
That is right! I must have missed that :)

File: src/main/java/generators/ItemGenerator.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        Bag vintageCourier = new Bag(BagType.HANDBAGS, "Vintage courier bag", 25, "NK022", 60.00d);
+        Bag chicVintage = new Bag(BagType.HANDBAGS, "Chic vintage DeVille", 14, "NK035", 78.00d);
+        Bag vintageExotik = new Bag(BagType.HANDBAGS, "Vintage exotik carry bag", 12, "NK125", 62.00d);
+        Bag multiUseHandBag = new Bag(BagType.HANDBAGS, "Multi use hand bag", 10, "NK028", 65.00d);
+        Bag vintageBag = new Bag(BagType.HANDBAGS, "Vintage bag", 18, "NK089", 69.00d);
+        Bag vintageChic = new Bag(BagType.HANDBAGS, "Vintage chic handbag", 15, "NK019", 68.00d);
+        Bag vintageBeachBag = new Bag(BagType.BEACH_BAGS, "Vintage chic handbag", 15, "NK095", 60.00d);
+        Bag retroStyleBag = new Bag(BagType.BEACH_BAGS, "Retro style bag", 20, "NK071", 48.00d);
+        Bag vintageLaptopBag = new Bag(BagType.LAPTOP_BAGS, "Vintage laptop bag", 24, "NK124", 78.00d);
+        Bag vintageBagWithLeatherBands = new Bag(BagType.BAGS, "Vintage bag with leather bands", 9, "NK033", 68.00d);
```
Comment (katarzynaboguslawska) [inline src/main/java/generators/ItemGenerator.java]: I'd probably create the list of bags outside the method as a field of the class, to keep the responsibilities of the method to minimum

**Comment by agaslimak** [comment]
it looks fine to me

**Comment by agaslimak** [comment]
Bag model with Bag Random Generator and Payment generator

**Pull request merged**